docker/convert/convert.py

```diff
@@ -0,0 +1,34 @@
+import sys
+import os
+import errno
+import recordio
+import paddle.v2.dataset as ds
+
+def mkdir_p(path):
+    try:
+        os.makedirs(path)
+    except OSError as exc:  # Python >2.5
+        if exc.errno == errno.EEXIST and os.path.isdir(path):
+            pass
+        else:
+            raise
+
+def convert(output_path, name):
+    print "proc " + name
+    mod = __import__("paddle.v2.dataset." + name, fromlist=[''])
+
+    path = output_path + "/" + name
+    mkdir_p(path)
+
+    mod.convert(path)
```

Comment:
I saw the convert function here:https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/v2/dataset/common.py#L167, it also needs some other required parameters such as reader and num_shards. So maybe the code can not be run correctly, or I missed sth. ?